gateway/mw_go_plugin.go

```diff
@@ -236,10 +246,10 @@ func (m *GoPluginMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Reque
 	return
 }
 
-// goPluginFromTykVersion builds a name of plugin based on tyk version
+// getGoPluginNameFromTykVersion builds a name of plugin based on tyk version
 // os and architecture. The structure of the plugin name looks like:
 // {plugin-dir}/{plugin-name}_{GW-version}_{OS}_{arch}.so
-func (m *GoPluginMiddleware) goPluginFromTykVersion(version string) string {
+func (m *GoPluginMiddleware) getGoPluginNameFromTykVersion(version string) string {
```

Comment:
@sredxny why are you adding Go to the func name? it's already a function for `GoPluginMiddleware`

Reply:
ahhhh makes sense!